client-app/shared/wishlists/components/wishlist-dropdown-menu.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <VcDropdownMenu :y-offset="4" :x-offset="0" placement="bottom-end">
+    <template #trigger>
+      <VcButton icon="cog" color="secondary" variant="outline" size="xs" />
+    </template>
+    <template #content>
+      <VcMenuItem color="secondary" @click="$emit('edit')">
+        <VcIcon name="pencil" />
+
+        <span>{{ $t("shared.wishlists.list_card.list_edit_button") }}</span>
+      </VcMenuItem>
+      <VcMenuItem
+        v-if="isCorporateMember && scopeButton"
+        :color="scopeButton.color"
+        @click="$emit('setScope', scopeButton.scope)"
+      >
+        <VcIcon :name="scopeButton.icon" />
```

Comment:
use class="..." to change icon color
For example:
class="text-[--color-accent-600]"